### DISCUSSION TITLE
How do you calculate the position?

### DISCUSSION THREAD

**Discussion by RahimMahmoudzadeh** [readium/kotlin-toolkit#102] [Q&A] [answered]
Greetings 
I wanted to know how do you calculate the position?
If it is word based, how many words do you consider a position?
thank you

**Comment by mickael-menu** [comment, marked as answer]
It depends on the publication media type:

* For a PDF or a comic book, one page = one position.
* For an EPUB fixed layout, one page/resource = one position.
* For audiobooks, we don't have positions.

And I guess you are interested about EPUB reflowable, in this case by default we use the same technique as Adobe RMSDK : the ZIP archive entry length divided by 1024 gives the number of positions in a resource. There's no processing of the text content/words.

More info :
* https://github.com/readium/architecture/issues/101
* https://github.com/readium/architecture/issues/123
